tests/models/opt/test_modeling_opt.py

```diff
@@ -307,16 +307,15 @@ def test_logits(self):
         model = OPTForCausalLM.from_pretrained(self.path_model)
         model = model.eval()
         tokenizer = GPT2Tokenizer.from_pretrained(self.path_model)
-        tokenizer.add_special_tokens({"pad_token": "<pad>"})
 
         prompts = [
             "Today is a beautiful day and I want to",
             "In the city of",
             "Paris is the capital of France and",
             "Computers and mobile phones have taken",
         ]
-        input_ids = tokenizer(prompts, return_tensors="pt", padding=True).input_ids
-        logits = model(input_ids)[0].mean(dim=-1)
+        inputs = tokenizer(prompts, return_tensors="pt", padding=True, add_special_tokens=False)
```

Comment:
to be identical to metaseq let's not append the BOS token

Reply:
As usual, I would like very much this comment is added to the code :-)